app/_locales/vi/messages.json

```diff
@@ -2089,6 +2079,9 @@
   "newNetworkAdded": {
     "message": "“$1” đã được thêm thành công!"
   },
+  "newNftAddedMessage": {
+    "message": "Bộ sưu tập đã được thêm thành công!"
```

Comment:
Similar to above — "NFT" is missing here, but I'm not sure what the correct translation is.